themes/mongocli/page.html

```diff
@@ -44,7 +44,8 @@
       <span class="alert-message">
         <h4 class="alert-heading" style="margin-bottom: 10px;">Alpha
         Release of MongoDB CLI</h4>
-        This is a pre-release early version of MongoDB CLI. Don’t use MongoDB CLI in production environments.
+        This is a pre-release early version of MongoDB CLI. Don’t use
```

Comment:
Quibble: I'd go with "do not" rather than the contraction.

Reply:
I had someone else's feedback to use the contraction, haha. But I'm with you, I'm changing it back. ;)